components/script/script_thread.rs

```diff
@@ -2073,12 +2075,17 @@ impl ScriptThread {
         window.evaluate_media_queries_and_report_changes();
     }
 
-    /// Initiate a non-blocking fetch for a specified resource. Stores the InProgressLoad
+    /// Sends a message to constellation to check for cross-origin redirects. Stores the InProgressLoad
```

Comment:
`/// Instructs the constellation to fetch the document that will be loaded. Stores the ...`